src/common/types/uuid.cpp

```diff
@@ -222,4 +222,35 @@ hugeint_t UUIDv7::GenerateRandomUUID() {
 	return GenerateRandomUUID(engine);
 }
 
+hugeint_t BaseUUID::FromBlob(const_data_ptr_t input) {
+	// Based on UUIDValueConversion::ReadParquetUUID from parquet extension
+	hugeint_t result;
+	result.lower = 0;
+	uint64_t unsigned_upper = 0;
+	for (idx_t i = 0; i < sizeof(uint64_t); i++) {
+		unsigned_upper <<= 8;
```

Comment:
We could probably use BSwap (duckdb/common/bswap.hpp)

But since this is (I assume?) directly copied from existing code, it's fine to leave it as-is